src/collection-browser.ts

```diff
@@ -513,8 +522,13 @@ export class CollectionBrowser
   }
 
   handleResize(entry: ResizeObserverEntry): void {
+    const previousView = this.mobileView;
     if (entry.target === this.contentContainer) {
-      this.mobileView = entry.contentRect.width < 600;
+      this.mobileView = entry.contentRect.width < this.mobileBreakpoint;
+      // If changing from desktop to mobile disable transition
+      if (this.mobileView && !previousView) {
+        this.isResizeToMobile = true;
```

Comment:
Q: when does `this.isResizeToMobile` turns to false?

Reply:
376, 380
```          @click=${() => {
            this.isResizeToMobile = false;
```